cmd/frontend/internal/auth/userpasswd/reset_password.go

```diff
@@ -76,6 +78,61 @@ func HandleResetPasswordInit(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+var setPasswordEmailTemplates = txemail.MustValidate(txtypes.Templates{
+	Subject: `Set your Sourcegraph password`,
+	Text: `
+Your administrator created an account for you on Sourcegraph.
+
+To set the password for {{.Username}} on Sourcegraph, follow this link:
+
+  {{.URL}}
+`,
+	HTML: `
+<p>
+  Your administrator created an account for you on Sourcegraph.
+</p>
+
+<p><strong><a href="{{.URL}}">Set password for {{.Username}}</a></strong></p>
+`,
+})
+
+// Send the password reset email directly to the user on user creation by admin
```

Comment:
```suggestion
// HandleSetPasswordEmail sends the password reset email directly to the user for users created by site admins.
```

qq: every other place is using the tone of "reset password", but the email body and this function is using the tone of "set password", could we be more consistent here?

Reply:
The reason I used set password is because this email is only sent when the user is setting their initial password. While it's technically a password reset from the automatically generated password, I'd argue it's going to look more like setting one's password from the user's perspective.